src/platform/datapath_raw_socket.c

```diff
@@ -43,19 +59,8 @@ CxPlatGetSocket(
     return Socket;
 }
 
-_IRQL_requires_max_(PASSIVE_LEVEL)
-void
-QuicCopyRouteInfo(
-    _Inout_ CXPLAT_ROUTE* DstRoute,
-    _In_ CXPLAT_ROUTE* SrcRoute
-    )
-{
-    CxPlatCopyMemory(DstRoute, SrcRoute, (uint8_t*)&SrcRoute->State - (uint8_t*)SrcRoute);
-    CxPlatUpdateRoute(DstRoute, SrcRoute);
-}
-
 void
-CxPlatResolveRouteComplete(
+RawResolveRouteComplete(
```

Comment:
Generally, I'd prefer to have either `CxPlat` prefix all functions (usually only for "public" functions to upper layer) or something like `Datapath` or the object type.